docs/updates.md

```diff
@@ -2,6 +2,11 @@
 
 Here are some updates we've made to our autograder platform. If you have any questions or issues with any of these changes, please email [help@gradescope.com](mailto:help@gradescope.com).
 
+## Nov 8th, 2022
+
+We have upgraded `openssl` to 3.0.2-0ubuntu1.7 on Ubuntu 22.04 autograder images.
+See https://ubuntu.com/security/notices/USN-5710-1
```

Comment:
Seems like we might need to add some link Markdown syntax to make it into a link: https://gradescope-autograders--87.org.readthedocs.build/en/87/updates/

```suggestion
See [https://ubuntu.com/security/notices/USN-5710-1](https://ubuntu.com/security/notices/USN-5710-1) for more information.
```